tests/cross_fw/install/test_install.py

```diff
@@ -121,7 +121,8 @@ def test_install_with_tests_requirements(tmp_path: Path,
             pytest.xfail('Disabled until NNCF with torch version supporting CUDA 11.6 backend is exposed in a release')
         venv_path = create_venv_with_nncf(tmp_path, package_type, venv_type, extra_reqs={backend})
         pip_with_venv = f'. {venv_path}/bin/activate && {venv_path}/bin/pip'
-        subprocess.call(
-            f'{pip_with_venv} install -r {PROJECT_ROOT}/tests/{backend}/requirements.txt',
+        backend_name = 'tensorflow' if backend == 'tf' else backend            
```

Comment:
Why do we have such a duality in the backend name? 
In which cases is it `tensorflow` and in which is `tf`?

Reply:
Historically, 'tf' is extra parameter for installation via pip but all folders are named as `tensorflow`.

Reply:
Backend should be always `tf`. Why do you need `else`?
https://github.com/openvinotoolkit/nncf/blob/develop/tests/cross_fw/install/conftest.py#L16